tests/unit/src/test/scala/tests/imports/DependencySuite.scala

```diff
@@ -84,16 +84,16 @@ class DependencySuite extends BaseMarkdownSuite {
        |  not found: https://repo1.maven.org/maven2/org/scalameta/foobar/1.2.1/foobar-1.2.1.pom
        |import $dep.`org.scalameta::mmunit:2.3.4`, $dep.`org.scalameta:foobar:1.2.1`
        |                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-       |error: dep-error.md:4:13: Error downloading org.scalameta:not-exists_2.13.2:2.3.4
-       |<redacted user.home>
-       |  not found: https://repo1.maven.org/maven2/org/scalameta/not-exists_2.13.2/2.3.4/not-exists_2.13.2-2.3.4.pom
-       |import $dep.`org.scalameta:::not-exists:2.3.4`
-       |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        |error: dep-error.md:3:13: Error downloading org.scalameta:mmunit_2.13:2.3.4
        |<redacted user.home>
        |  not found: https://repo1.maven.org/maven2/org/scalameta/mmunit_2.13/2.3.4/mmunit_2.13-2.3.4.pom
        |import $dep.`org.scalameta::mmunit:2.3.4`, $dep.`org.scalameta:foobar:1.2.1`
        |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+       |error: dep-error.md:4:13: Error downloading org.scalameta:not-exists_2.13.4:2.3.4
```

Comment:
I'm not 100% sure why the order flips here between 2.13.2 and 2.13.4 🤔

Reply:
I think it's ok, the order should not matter here.